testing-overhead/src/test/java/io/opentelemetry/agents/AgentResolver.java

```diff
@@ -8,8 +8,8 @@
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
-import java.nio.file.StandardCopyOption;
 import java.nio.file.StandardOpenOption;
+import java.nio.file.StandardCopyOption;
```

Comment:
That is certainly not an alphabetical order :)

Reply:
🙈 
This is the best comment I will read this year.  😝